frontend/packages/console-plugin-sdk/src/typings/notification-alerts.ts

```diff
@@ -0,0 +1,24 @@
+import { Extension } from './base';
+import { Alert } from '@console/internal/components/monitoring/types';
+
+namespace ExtensionProperties {
+  export interface AlertAction {
+    /* Alert name for the key in alertAction Map */
+    alert: string;
+    /* Value for the alertAction Map */
+    value: {
```

Comment:
nicer interface would be without `value` (which is again implementation detail which dont need to be reflected into interface)
```typescript
export interface AlertAction {
  /* Alert name as defined by `alert.rule.name` property */
  alert: string;

  /* Action text */
  text: string;
  
  /* Action href link */
  path: (alert: Alert) => string;
};
```